internal/bitwarden/bw/client.go

```diff
@@ -67,23 +68,48 @@ func DisableSync() Options {
 }
 
 func (c *client) CreateObject(obj Object) (*Object, error) {
-	objEncoded, err := c.encode(obj)
-	if err != nil {
-		return nil, err
-	}
+	var out []byte
+	var err error
+	if string(obj.Object) == "attachment" {
+		out, err = c.cmdWithSession("create", string(obj.Object), "--file", obj.File, "--itemid", obj.ItemId).RunCaptureOutput()
+		if err != nil {
+			return nil, err
+		}
+		var tmpObj Object
+		err = json.Unmarshal(out, &tmpObj)
+		if err != nil {
+			return nil, err
+		}
+		result := filterAttachments(tmpObj.Attachments, func(val Attachment) bool {
+			return val.FileName == obj.FileName
+		})
+		if len(result) > 0 {
```

Comment:
If we don't try to fit an attachment into an object, this portion of the code can go away as we will return the object and determine the attachment's ID in the caller. Ultimately, the only difference between `CreateObject` and `CreateItemAttachment` would be its parameters. I was wondering if it still makes sense to pass an `Object` as parameter but I didn't look into it.